index.php

```diff
@@ -27,11 +27,11 @@
 
 
       <?php 
-      if( ! empty($right_column) ) {
+      if( !empty( $left_column ) || ( is_single() && in_cat_ancestor_of( 'laboratorio' ) )  ) {
       ?>
         <div class="sidebar col col-3">
 
-        <?php echo $right_column; ?> 
+        <?php echo $left_column; ?> 
```

Comment:
this still needs an empty check conditional. and it needs to be below the index menu